barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_rollup_recursive_flavor.hpp

```diff
@@ -115,6 +115,62 @@ template <typename BuilderType> class UltraRollupRecursiveFlavor_ : public Ultra
             }
         }
 
+        /**
+         * @brief Serialize verification key to field elements.
+         *
+         * @return std::vector<FF>
+         */
+        std::vector<FF> to_field_elements() const
+        {
```

Comment:
please, add a sentence as to why this is needed (here and in the other places where this method was added)